mocha-casperjs.js

```diff
@@ -47,8 +47,13 @@ module.exports = function (Mocha, casper, utils) {
     })
   })
 
+  function clearSteps() {
+    casper.step = 0
+    casper.steps = []
+  }
```

Comment:
Can we just create new instance of casper for every test? In this case we will not change something inside of casper and from the other side will fix problem with steps skipping.